plans/install.pp

```diff
@@ -94,7 +105,7 @@
   $master_pe_conf = epp('pe_xl/master-pe.conf.epp',
     console_password       => $console_password,
     master_host            => $master_host,
-    puppetdb_database_host => $puppetdb_database_host,
+    puppetdb_database_host => "$puppetdb_database_host",
```

Comment:
This parameter has to be a string. So, database_target fails because it is an array.  It needs to be quoted in order to pass an empty string if it is undefined, otherwise it fails because the undef is passed rather than a string.

Reply:
I was wondering this as well. Could you please add a comment with this explanation?

Reply:
Comment added to clarify the purpose of the quotes. Thanks.